deployment/cloudwatch-agent-config.json

```diff
@@ -0,0 +1,54 @@
+{
+  "agent": {
+    "metrics_collection_interval": 60,
+    "run_as_user": "root"
+  },
+  "logs": {
+    "logs_collected": {
+      "files": {
+        "collect_list": [
+          {
+            "file_path": "/home/ubuntu/db-service/logs/info.log*",
+            "log_group_name": "staging-dbservice",
```

Comment:
this log group is for staging. Where is the log group for prod?

Reply:
Updated in `cloudwatch.tf` file. No need of this config anymore